index.html

```diff
@@ -3,17 +3,68 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css/index.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <title>Karma</title>
 </head>
 <body>
-
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
-
+  <!-- I have deliberately completed this WITHOUT any IDs or Classes for Specificity/Selector Practise -->
+  <header>
+    <a href="./index.html"><img src="./img/karma-logo.svg" alt="Karma Logo"></a>
```

Comment:
alt text is tricky, I try to keep in mind the intension of the element. In this case when an assistive tech user hears a list of links, "Karma Logo" is ambiguous "Karma home page" would be better